spec/models/automated_checks/claim_verifiers/census_subjects_taught_spec.rb

```diff
@@ -113,6 +124,10 @@ module ClaimVerifiers
                     expect(subject).to eq("[School Workforce Census] - Eligible:\n<pre>Subject 1: Mathematics / Mathematical Development (Early Years)\nSubject 2: Statistics\n</pre>\n")
                   end
 
+                  it "returns 'Eligible' with the school workforce census subjects", if: policy == LevellingUpPremiumPayments do
```

Comment:
What's this `if` thing? I've not seen that before

Reply:
It's a conditional `if`, because these specs run in an `each` iteration. I copied the format from the other policies.